rtCamp/web-stories-wp
# Use renderer classes to render the blocks.

## Summary

This PR updates the blocks to use Story_Renderer class to output the block. Along with few fixes required to make the blocks work properly with the new php-api changes. Also this PR addresses the lightbox feature for the stories.

## Relevant Technical Choices

- For non-AMP pages.
-- Use custom JS to show stories in single lightbox.

## To-do

- Need to use the single lightbox/story-player for AMP pages.

## User-facing changes

n/a

## Testing Instructions

- Select the Latest/Selected Web Stories block.
- Check the FrontEnd with different views selected on AMP and non-AMP pages.

---



Fixes #79
